### PR TITLE
Fix no result for anomalies with one day data

### PR DESCRIPTION
**Pull request by IDoneShaveIt**
Fix no result for anomalies with one day data.
This is how it will look at the UI:
image

### CODE REVIEW

File: macros/edr/data_monitoring/anomaly_detection/get_anomaly_scores_query.sql [hunk 1]
```diff
@@ -113,6 +113,7 @@
                 column_name,
                 metric_name,
                 case
+                    when training_stddev is null then null
```
Comment (Maayan-s) [inline macros/edr/data_monitoring/anomaly_detection/get_anomaly_scores_query.sql:116]: Make sure this works for all platforms, we might need to pass null casted as int

File: macros/edr/tests/on_run_end/handle_tests_results.sql [hunk 2]
```diff
@@ -149,6 +150,13 @@
              {%- endif %}
         order by bucket_end
     {%- endset -%}
+    {% set test_results_description %}
+        {% if elementary.insensitive_get_dict_value(anomaly_dict, 'anomaly_score') is none %}
+            Could not calculate anomaly score for this test due to lack of data.
```
Comment (Maayan-s) [inline macros/edr/tests/on_run_end/handle_tests_results.sql:155]: "Not enough data to calculate anomaly score."

File: models/edr/data_monitoring/anomaly_detection/metrics_anomaly_score.sql [hunk 3]
```diff
@@ -55,8 +55,9 @@ metrics_anomaly_score as (
         dimension_value,
         metric_name,
         case
-           when training_stddev = 0 then 0
-           else (metric_value - training_avg) / (training_stddev)
+            when training_stddev is null then null
```
Comment (Maayan-s) [inline models/edr/data_monitoring/anomaly_detection/metrics_anomaly_score.sql:58]: Same comment about null

**Review by Maayan-s — CHANGES_REQUESTED** [review]
Verify Redshift + change desc

File: macros/edr/data_monitoring/anomaly_detection/get_anomaly_scores_query.sql (hunk 1 above)
Comment (IDoneShaveIt) [reply macros/edr/data_monitoring/anomaly_detection/get_anomaly_scores_query.sql:116]: Working on all platforms 🙂

File: models/edr/data_monitoring/anomaly_detection/metrics_anomaly_score.sql (hunk 3 above)
Comment (IDoneShaveIt) [reply models/edr/data_monitoring/anomaly_detection/metrics_anomaly_score.sql:58]: Working on all platforms 🙂

**Pull request merged**